### DISCUSSION TITLE
Setup goals on "Add a website"

### DISCUSSION THREAD

**Discussion by davidmondok** [plausible/analytics#3256] [Anything Else]
It's a rather simple feature request that would save us time and also prevent us forgetting to create these three commonly used goals:

- 404
- Outblound links
- File downloads

It would be great to add three additional checkboxes to the "Your website details" page (the one we see after clicking "+ Add a website") which set up the goals automatically if checked.

**Comment by metmarkosaric** [comment]
good points @davidmondok! both of these is something we'd like to improve in the future (showing you the exact snippet according to your needs and automatically adding the goals for you)

**Reply by davidmondok** [reply]
Good to hear. To be honest, the checkboxes are not even needed. You could just create the goal automatically the first time you receive them. It doesn't really make sense having to create a goal for them. But I guess this is a deeper change than one might assume :)
